### PR TITLE
Vagelim/kafka yaml

### PR DESCRIPTION
**Pull request by vagelim**
cc: @olivielpeau

### CODE REVIEW

File: conf.d/kafka.yaml.example [hunk 1]
```diff
@@ -17,15 +21,161 @@ instances:
...
+            metric_type: gauge
+            alias: kafka.producer.io_wait
+
+
+    # v0.8.2.x Consumers
+    - include:
+        domain: 'kafka.consumer'
+        bean_regex: 'kafka\.consumer:type=FetcherLagMetrics,name=ConsumerLag,clientId=.*'
+        attribute:
+            alias: kafka.consumer.lag
```
Comment (olivielpeau) [inline conf.d/kafka.yaml.example]: I think you forgot to fully specify the attribute here

Comment (irabinovitch) [inline conf.d/kafka.yaml.example]: Do we need to make these changes in the tile and docs as well?

**Comment by olivielpeau** [comment]
@evan: added a comment.

As @irabinovitch has said it would make sense to add these metrics in the metrics metadata (some of the metrics depend on the version of Kafka but I think it's ok to add them all)

**Comment by vagelim** [comment]
@olivielpeau it seems that if you do not specify a metric_type, it defaults to gauge: http://docs.datadoghq.com/integrations/java/

@irabinovitch will update metadata accordingly

**Comment by vagelim** [comment]
@olivielpeau that metric was a duplicate; removed

@irabinovitch I will update the metadata accordingly

**Comment by olivielpeau** [comment]
Implemented the diff between this PR and #2371 (and fixes the change in #2425) in #2435

**Pull request closed without merging**